docs/reference/jspsych-metadata.md

```diff
@@ -0,0 +1,156 @@
+# Metadata Module
+
+The metadata module contains functions for interacting metadata according to [Psych-DS standards](https://psych-ds.github.io/). To interact with the metadata, we strongly recommend you call the generate method on the experiment data then adjust fields accordingly. The generate method uses documentation for plugins and extensions created in the main JsPsych repo to create default descriptions. This works best for experiments run in v8+, but will also work for old experimental data.
+
+---
+
+## metadata.generate
+
+```javascript
+var metadata = new jsPsychMetadata();
+metadata.generate(properties);
+```
+
+### Parameters
+
+| Parameter | Type | Description
+----------|------|------------
+| data      | object, string | Can be passed|
+| metadata  | object | Optional metadata that can be passed in to adjust the output of the generate method. Each key-value pair will map to a parameter field, and will either overwite existing data or create new fields. All normal fields can be adjusted as entries within the object, but for authors and variables the entries need to be nested within an "author" or "variables" key mapping. Similarily, updating the descriptions of variables needs to be a nested map. |
```

Comment:
I think this needs more to explain. Perhaps a pointer to another section of the doc?